Makefile

```
@@ -45,8 +45,11 @@ layers: docker-images
 			echo "### Exporting $${dir} PHP$${php_version}"; \
 			echo "###"; \
 			cd ${PWD} ; rm -rf export/tmp/${layer} || true ; cd export/tmp ; \
-			docker run --entrypoint "tar" bref/$${dir}-php-$${php_version} -ch -C /opt . | tar -x ; \
-			zip --quiet -X --recurse-paths ../`echo "$${dir}-php-$${php_version}" | sed -e "s/layers\//layer-/g"`.zip . ; \
+			CID=$$(docker create --entrypoint=/bin/bash bref/$${dir}-php-$${php_version}) ; \
```

As we don't run the container entrypoint definition seems useless unless it's required by docker create, no?
```suggestion
			CID=$$(docker create bref/$${dir}-php-$${php_version}) ; \
```

---

https://github.com/brefphp/extra-php-extensions/pull/82#discussion_r505447666